.storybook-s2/docs/typography.js

```diff
@@ -0,0 +1,37 @@
+import { style } from '../../packages/@react-spectrum/s2/style/spectrum-theme' with {type: 'macro'};
+import {Link} from '@react-spectrum/s2';
+
+export function H2({children}) {
+  let id = anchorId(children);
+  return <h2 className={style({fontSize: 'heading-xl', lineHeight: 'heading', color: 'heading', marginTop: 48, marginBottom: 24})} id={id}>{children}</h2>
+}
+
+export function H3({children}) {
+  let id = anchorId(children);
+  return <h3 className={style({fontSize: 'heading', lineHeight: 'heading', color: 'heading', marginTop: 32, marginBottom: 16})} id={anchorId(children)}>{children}</h3>
+}
+
+export function H4({children}) {
+  let id = anchorId(children);
+  return <h4 className={style({fontSize: 'heading-sm', lineHeight: 'heading', color: 'heading', marginTop: 32, marginBottom: 8})} id={anchorId(children)}>{children}</h4>
```

Comment:
I was borrowing this file and noticed that we declare the `id` and don't use it, except for in the `H2`. The rest call `anchorId` inline for the `id` prop.